environments/skeleton/{{cookiecutter.environment}}/tofu/control.tf

```diff
@@ -11,7 +11,8 @@ resource "openstack_networking_port_v2" "control" {
   admin_state_up = "true"
 
   fixed_ip {
-    subnet_id = data.openstack_networking_subnet_v2.cluster_subnet[each.key].id
+    subnet_id  = data.openstack_networking_subnet_v2.cluster_subnet[each.key].id
+    ip_address = (var.control_ip_addresses != {})? var.control_ip_addresses[each.key]: null
```

Comment:
This resource has a for_each parameter. So the particular resource instance (i.e, port) for each item in that for_each has a has a _single_ ip address, not a list. Else you're trying to assign multiple IPs from different subnets to _each_ port.